packages/opentelemetry-types/src/metrics/Meter.ts

```diff
@@ -14,30 +14,31 @@
  * limitations under the License.
  */
 
-import { SpanContext } from '../trace/span_context';
-import { DistributedContext } from '../distributed_context/DistributedContext';
-import { Measure, MeasureOptions, Measurement } from './measure';
-import { Metric, MetricOptions } from './metric';
-import { CounterTimeseries } from './counter';
-import { GaugeTimeseries } from './gauge';
-
-export interface RecordOptions {
-  // spanContext represents a measurement exemplar in the form of a SpanContext.
-  spanContext?: SpanContext;
-  // distributedContext overrides the current context and adds dimensions
-  // to the measurements.
-  distributedContext?: DistributedContext;
-}
+import { Metric, MetricOptions } from './Metric';
+import { CounterHandle, GaugeHandle } from './Handle';
+import { MeasureOptions, Measure } from './Measure';
 
+/**
+ * An interface to allow the recording metrics.
+ *
+ * {@link Metric}s are used for recording pre-defined aggregation (Gauge and
+ * Counter), or raw values ({@link Measure}) in which the aggregation and labels
+ * for the exported metric are deferred.
+ */
 export interface Meter {
-  // Creates and returns a new @link{Measure}.
+  /**
+   * Creates and returns a new {@link Measure}.
+   * @param name the name of the metric.
+   * @param [options] the measure options.
+   */
   createMeasure(name: string, options?: MeasureOptions): Measure;
```

Comment:
I took a stab at updates the other day, take a look at https://github.com/bg451/opentelemetry-node/pull/5. I think this should return `Metric<MeasureHandle>`, though I can apply the changes in a separate PR

Reply:
bg451#5 lgtm, I will merge this one and wait for your PR for the next revision of metrics API (including counter  -> cumulative).